package.json

```diff
@@ -5,7 +5,8 @@
   "main": "bindings/node",
   "scripts": {
     "test": "tree-sitter test",
-    "install": "node-gyp rebuild"
+    "generate": "tree-sitter generate",
+    "compile": "node-gyp rebuild"
```

Comment:
`node-gyp rebuild` has a hard dependency on the `generate` step -- it blows up without the src/parser.c and bindings/node/binding.cc; it could `npm run generate && node-gyp rebuild` maybe?

we also do need to have the `install` lifecycle script step generate the artifacts and rebuild if they're no longer packaged for downstream dependencies